demo/slimfaas-ingress.yml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: slimfaas
+  namespace: slimfaas-demo
+spec:
+  rules:
+  - host: localhost
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: slimfaas
+            port:
+              number: 5000
```

Comment:
Demo does not work with that mode.
I do not know how work ingress, is it possible to bind 30021 (public port for the demo) to redirect to the internal 5000 port of SlimFaas?

Reply:
True, this only opens ports for internal communication between SlimFaaS and worker containers, but you'd also need to bind a host port to be able to call SlimFaaS. Since you could either use a NodePort or Ingress for it, I've left that out of this manifest and propose to use either of the next two manifests.